src/main/java/org/marvelution/jji/security/SyncTokenAuthenticationFilter.java

```diff
@@ -84,7 +85,7 @@ protected void doFilterInternal(
     {
         try
         {
-            JWTClaimsSet claimsSet = tokenAuthenticator.authenticate(new SyncTokenRequest(request));
+            JWTClaimsSet claimsSet = tokenAuthenticator.authenticate(new SyncTokenRequest(HttpServletRequestWrapper.fromJakartaHttpServletRequest(request)));
```

Comment:
Temporary until `SyncTokenRequest` is migrated from EE 8 to EE 9, at which point this compatibility code can be deleted.

Reply:
This is a shared library with Atlassian Jira, and they haven't yet adopted EE9 so this may take a while.
Good thing Jenkins has a compatibility util.